config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -1,3850 +0,0 @@
-# Autogenerated by genjobs.go, do NOT edit!
```

Comment:
Looks like something is wrong with this PR - this file shouldn't be removed...

Reply:
Yeah, I didn't realize that /hack/update-config.sh failed and only removed the file without writing an update one. Now should be good